packages/@react-native-windows/cli/src/runWindows/utils/deploy.ts

```diff
@@ -275,45 +251,69 @@ export async function deployToDesktop(
     args.push('--direct-debugging', options.directDebugging.toString());
   }
 
-  await runPowerShellScriptFunction(
-    'Removing old version of the app',
-    windowsStoreAppUtils,
-    `Uninstall-App ${appName}`,
-    verbose,
-  );
-
   await runPowerShellScriptFunction(
     'Enabling Developer Mode',
     windowsStoreAppUtils,
     'EnableDevMode',
     verbose,
   );
 
+  const appPackageFolder = getAppPackage(options, projectName);
+
   if (options.release) {
     await runPowerShellScriptFunction(
-      'Installing new version of the app',
+      'Removing old version of the app',
       windowsStoreAppUtils,
-      `Install-App "${script}" -Force`,
+      `Uninstall-App ${appName}`,
       verbose,
     );
-  } else {
-    // Install the app package's dependencies before attempting to deploy.
+    const script = glob.sync(
+      path.join(appPackageFolder, 'Add-AppDevPackage.ps1'),
+    )[0];
     await runPowerShellScriptFunction(
-      'Installing dependent framework packages',
+      'Installing new version of the app',
       windowsStoreAppUtils,
-      `Install-AppDependencies ${appxManifestPath} ${appPackageFolder} ${options.arch}`,
+      `Install-App "${script}" -Force`,
       verbose,
     );
-    await build.buildSolution(
-      buildTools,
-      slnFile,
-      'Debug',
-      options.arch,
-      {DeployLayout: 'true'},
-      verbose,
-      'deploy',
-      options.buildLogDirectory,
+  } else {
+    // If we have DeployAppRecipe.exe, use it (start in 16.9 Preview 2, don't use 16.8 even if it's there as that version has bugs)
+    const appxRecipe = path.join(
+      path.dirname(appxManifestPath),
+      `${projectName}.build.appxrecipe`,
     );
+    const ideFolder = `${buildTools.installationPath}\\Common7\\IDE`;
+    const deployAppxRecipeExePath = `${ideFolder}\\DeployAppRecipe.exe`;
+    if (
+      vsVersion.gte(Version.fromString('16.9.30801.93')) &&
```

Comment:
Fixed version is backported to 16.8.4, so correct version to compare against is 16.8.30906.45

Reply:
```suggestion
      vsVersion.gte(Version.fromString('16.8.30906.45')) &&
```

Reply:
Technically, 16.9 Preview 1 is also disqualified, but it is unlikely that someone would actually run it (given that we are at Preview 4 now)

Reply:
yeah I think it's ok to have it not work in 16.9 Preview 1. As long as we support the right public releases we should be good